src/cljs/org/broadinstitute/firecloud_ui/persistence.cljs

```diff
@@ -7,8 +7,8 @@
 (defn- generate-persistence-key [key]
   (keyword (str "state:" @utils/current-user ":" key)))
 
-(defn save [{:keys [key state except]}]
-  (utils/local-storage-write (generate-persistence-key key) (apply dissoc @state except)))
+(defn save [{:keys [key state except only]}]
+  (utils/local-storage-write (generate-persistence-key key) (apply dissoc (if only (select-keys @state only) @state) except)))
```

Comment:
Specifying both `:except` and `:only` seems nonsensical, so it might be good to guard against that with an assert (or a precondition?).  And we could clean up this line a bit too, something like:
```clojure
(utils/local-storage-write (generate-persistence-key key)
  (cond except (apply dissoc @state except)
        only (select-keys @state only)
        :else @state))
```